ccmlib/node.py

```diff
@@ -1417,6 +1417,16 @@ def __update_envfile(self):
 
         common.replace_in_file(conf_file, jmx_port_pattern, jmx_port_setting)
 
+
+        if common.is_win() and common.get_version_from_build(node_path=self.get_path()) >= '2.1':
```

Comment:
Same here -- there should be some documentation about the changes in C* that make this necessary.